pkg/controllermanager/apis/config/types.go

```diff
@@ -80,6 +84,17 @@ type ControllerManagerControllerConfiguration struct {
 	ManagedSeedSet *ManagedSeedSetControllerConfiguration
 }
 
+// BastionControllerConfiguration defines the configuration of the Bastion
+// controller.
+type BastionControllerConfiguration struct {
+	// ConcurrentSyncs is the number of workers used for the controller to work on
+	// events.
+	ConcurrentSyncs int
+	// MaxLifetime is the maximum time a Bastion resource can exist before it is
+	// forcefully deleted.
+	MaxLifetime time.Duration
```

Comment:
Most other fields use `metav1.Duration` instead of `time.Duration`:

```suggestion
	MaxLifetime metav1.Duration
```

(same for versioned type)